examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -1,5 +1,7 @@
-# This script adds metric reporting via a Determined core.Context. This allows you to view metrics in the WebUI.
-# In this example we report training and testing losses as metrics to the Determined master.
+# This script adds metric reporting via a Determined core.Context.
+# This allows you to view metrics in the WebUI.
+# In this example we report training and testing losses as metrics to
+# the Determined master.
```

Comment:
Please reword, first and last sentence seem awfully redundant.

(I know you only reflowed these lines and it's not really your fault)